attachment_zipped_download/tests/test_attachment_zipped_download.py

```diff
@@ -2,11 +2,26 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 import base64
 
-import odoo.tests
-from odoo.tests import new_test_user
+from odoo.exceptions import AccessError
+from odoo.tests import HttpCase, SavepointCase, new_test_user
 
 
-class TestAttachmentZippedDownload(odoo.tests.HttpCase):
+def create_attachment(env, user, name, model=False, res_id=False):
```

Comment:
Why doing this outside the test class? That way, you can't inherit it.

Reply:
To make it clear that we can use it from class / instance method and also because I've reused it in next PR #412 but your are right probably proprer to add a common TestCase. I'll refactor that a bit.

Reply:
Yes, the way to do it is to put a base test class without `test_*` methods, and in the same module, inherit that base class for the module tests, and you can use the base class in other dependent modules. Example:

https://github.com/OCA/bank-payment/blob/ededf771b02135ad23c888ed4103cf441bc6bc54/account_payment_order/tests/test_payment_order_inbound.py#L14
https://github.com/OCA/bank-payment/blob/ededf771b02135ad23c888ed4103cf441bc6bc54/account_payment_order_grouped_output/tests/test_payment_order_inbound_grouped.py#L12